docs/integrators/deep-history-squad.md

```diff
@@ -35,143 +39,326 @@ In the example above, the key `726573657276650000000a55544b2d326638306539` is de
 
 [comment]: # (mx-context-auto)
 
-## Public instance
+## MultiversX squad
 
-:::tip
-As of October 2022, a public deep-history squad isn't yet available. The instance is being prepared and should be ready in November 2022.
-:::
+The observing squads backing the public Gateways, in addition to being full history squads (serving past blocks, transaction and events up until the Genesis), also act as 3-epochs deep-history squads. That is, for **mainnet**, one can use https://gateway.multiversx.com to resolve historical account (state) queries, for the last 3 days. This interval is driven by the configuration parameter `[StoragePruning.NumEpochsToKeep]`, which is set to `4`, by default.
```

Comment:
serving past blocks, transaction**s** and events

Reply:
Fixed.